crates/oxc_hir/src/precedence.rs

```diff
@@ -24,7 +25,7 @@ impl<'a> GetPrecedence for Expression<'a> {
             Self::NewExpression(expr) => expr.precedence(),
             Self::CallExpression(expr) => expr.precedence(),
             Self::MemberExpression(expr) => expr.precedence(),
-            _ => Precedence::Lowest,
+            _ => Precedence::highest(),
```

Comment:
What is the reason for this change? @Boshen

Reply:
![image](https://github.com/Boshen/oxc/assets/1430279/16fb778e-3838-4b64-9256-083e79faf748)

https://tc39.es/ecma262/#sec-ecmascript-language-expressions